common/src/main/scala/org/apache/celeborn/common/protocol/message/ControlMessages.scala

```diff
@@ -548,6 +572,8 @@ object ControlMessages extends Logging {
         .setHighWorkload(highWorkload)
         .setWorkerStatus(PbSerDeUtils.toPbWorkerStatus(workerStatus))
         .setRequestId(requestId)
+        .setInternalPort(internalPort)
+        .setSecuredRpcPort(securedRpcPort)
         .build().toByteArray
```

Comment:
Should this also have .setSecuredPushPort(securedPushPort) and .setSecuredFetchPort(securedFetchPort)?

Reply:
Good catch! fixed.